docs/_templates/announcement-template.html

```diff
@@ -0,0 +1,10 @@
+<!-- This will appear in the announcement banner at the top
+It needs to be activated by passing the link in
+html_theme_options section of conf.py using the key: "announcement"
+-->
+<div class="sidebar-message">
+    This is a community-supported project.
```

Comment:
Do we want to add the survey text and link already?

Reply:
We could, I was sort of thinking of making this a placeholder and then making a survey PR just for better history

Reply:
Yeah thinking more, I think it's better we have a default/placeholder announcement and then we PR the survey one as needed--and we have something to go back to.

Reply:
ok sounds good!